### PR TITLE
merge User-Guides repository with meshcloud-docs repository

### PR DESCRIPTION
**Pull request by felixzieger**
After merging this PR, I will archive https://github.com/meshcloud/User-Guides/

### CODE REVIEW

File: docs/meshstack.how-to.create-project.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+id: meshstack.how-to.create-project
+title: How to create a meshProject
+---
+
+If you are not familiar with what a meshProject is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/meshcloud.index.html).
+
+## Pre-Requisites
+
+- Permissions: Your user needs either the Customer Admin or Customer Owner role in the meshCustomer you want to create the meshProject in
+- Valid Payment Method: meaning the payment method is not expired and assigned to your customer
```
Comment (Prerna21122112) [inline docs/meshstack.how-to.create-project.md:11]:
```suggestion
- Permissions: Your user needs either the Customer Admin or Customer Owner role in the meshCustomer for which you want to create the meshProject.
- Valid Payment Method: It means that the payment method is not expired and assigned to your customer
```

File: docs/meshstack.how-to.create-project.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ in the customer control plane under **Projects** > **Overview** (B).
+ ![Start meshProject Creation](./assets/project-creation/start-project-creation.png "Start Project Creation")
+A project creation wizard will appear.
+![Project Creation Wizard](./assets/project-creation/open-project-wizard.png "Project Creation Wizard").
+ 3. First you have to give your project a name. The best practice is to use the name of the application, product, or project plus the environment type. The identifier is automatically generated but can be modified by you. We recommend to keep the identifier as it is. ![General Information](./assets/project-creation/choose-project-name.png "Naming")
+ 4. The lower part of the first page provides you the possibility to tag your meshProject. Some tags are required to be specified by you. You can recognize the required tags by the red line on the left side of the input field. Some of the tags you chose might have an impact on the landing zone you are allowed to chose if a policy exists that might restricts the usage. Click next![Tags](./assets/project-creation/choose-tags.png "Tags")
```
Comment (Prerna21122112) [inline docs/meshstack.how-to.create-project.md:23]:
```suggestion
 4. The lower part of the first page allows you to tag your meshProject. Some tags are required to be specified by you. You can recognize the required tags by the red line on the left side of the input field. Some of the tags you choose might have an impact on the landing zone you are allowed to choose. This happens if there is a policy that might restrict the usage. Click next![Tags](./assets/project-creation/choose-tags.png "Tags")
```

File: docs/meshstack.how-to.create-project.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ 5. Select the payment method provided via the meshCustomer. Click next
+ 6. Now we come to the most important part. Choose at least one of the platforms for your future meshProject. The marketplace with interesting services provided by your organization is selected by default. If you want to have for example an AWS account pick AWS and select a provided meshLZ as well. Click next  ![Platform and Landing Zone](./assets/project-creation/choose-platform-lz.png "Platform and Landing Zone")
```
Comment (Prerna21122112) [inline docs/meshstack.how-to.create-project.md:25]: MeshLZ _ is this an okay term?

**Review by Prerna21122112 — APPROVED** [review]

File: docs/meshstack.how-to.confirm-project-deletion.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+3. Confirm the meshProject deletion by clicking on the bin icon. A pop-up for the meshProject deletion will be show a overview of the meshProject and the possibility to add a comment for the meshProject deletion. The checkbox is required to be checked to confirm the deletion.
+
+![Click the bin icon to confirm the meshProject deletion](./assets/partner/navigate-to-project-deletion2.png "Confirm meshProject deletion")
+![Review the Deletion request and give final meshProject deletion confirmation](./assets/partner/confirm-project-deletion.png "Confirm Deletion")
+
+## Optional
+
+You can also take other actions besides the meshProject deletion:
+
+- Project Histroy: View the meshProject History which contains all relevant logged information for the project like user assignments and meshTenant assignment
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.confirm-project-deletion.md:27]: f: Project History

File: docs/meshstack.how-to.manage-partner-level-permissions.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+- Permissions: Your user needs the Partner Admin role in the Partner Customer to assign other users administration and/or privileged roles
+
+## Step to Step Guide
+
+1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.
+![Select Admin Area in the top navigation bar](.././assets/partner/navigate-to-admin-area.png "Go to the Admin Area")
+2. You now need to go to the Partner Access Control where you can to onboard further privieleged users.
+![Select Partner Access Control menu item on the left side](../assets/partner/navigate-to-partner-access-control.png "Access Control")
+3. At the end of the `Assigned Users & Groups` list will be a input field. Type in the first-, last-name or email address to find and select the user you want to add. Choose a Partner or Privielged Role and press the `+` button.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.manage-partner-level-permissions.md:18]:
```suggestion
3. At the end of the `Assigned Users & Groups` list will be an input field. Type in the first-, last name or email address to find and select the user you want to add. Choose a Partner or a privileged Role and press the `+` button.
```

File: docs/meshstack.how-to.manage-partner-level-permissions.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## Pre-Requisites
+
+- Permissions: Your user needs the Partner Admin role in the Partner Customer to assign other users administration and/or privileged roles
+
+## Step to Step Guide
+
+1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.
+![Select Admin Area in the top navigation bar](.././assets/partner/navigate-to-admin-area.png "Go to the Admin Area")
+2. You now need to go to the Partner Access Control where you can to onboard further privieleged users.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.manage-partner-level-permissions.md:16]:
```suggestion
2. You now need to go to the Partner Access Control where you can onboard further privileged users.
```

File: docs/meshstack.how-to.onboard-team-to-project.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.how-to.onboard-team-to-project
+title: How to onboard your team to a meshProject
+---
+
+If you are not familiar with what a meshProject is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/meshcloud.index.html).
+
+## Pre-Requisites
+
+- Users you want to add to an meshProject needs to be added to the meshCustomer first. Check out the `Onboarding your team to your meshCustomer` guide.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-project.md:10]: t: would be nice to link the section

File: docs/meshstack.how-to.onboard-team-to-project.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.how-to.onboard-team-to-project
+title: How to onboard your team to a meshProject
+---
+
+If you are not familiar with what a meshProject is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/meshcloud.index.html).
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-project.md:6]: f: use relative Link

File: docs/meshstack.how-to.onboard-team-to-customer.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- At the end of the `Current Access` list will be a input field. Type in the first-, last-name or email address to find and select the user you want to add. Choose a Customer Role (Customer Owner, Admin or Employee) and press the `+` button.
+
+![Add a user to the customer](../assets/customer/customer-access-control-add-a-user.png "add a user")
+
+- Customer Owner can only be granted to max. 2 user per Customer. Also only a Customer Owner can grant another user the Owner role - except there is no Customer Owner at all.
+
+### Optional
+
+meshStack provides the optional configuration for 4-eyes access controls.
+Please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/meshcloud.customer.html#invite-users-to-a-meshcustomer-team).
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-customer.md:31]: f: use relative Link

File: docs/meshstack.how-to.onboard-team-to-customer.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
+---
+id: meshstack.how-to.onboard-team-to-customer
+title: How to onboard your team to your meshCustomer
+---
+
+If you are not familiar with what a meshCustomer is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/meshcloud.index.html).
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-customer.md:6]: f: use relative Link

File: docs/meshstack.how-to.onboard-team-to-customer.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+
+- Make sure you are in the meshCustomer you want to add further users. Do this by checking the drop-down in the upper-left corner.
+
+![Select meshCustomer in the upper left corner](../assets/customer/choose-customer.png "Pick meshCustomer")
+
+- Go to the access control panel by clicking on the `Access Control` tab
+
+![Click the Access Control tab](../assets/customer/customer-access-control.png "Access Control")
+
+- At the end of the `Current Access` list will be a input field. Type in the first-, last-name or email address to find and select the user you want to add. Choose a Customer Role (Customer Owner, Admin or Employee) and press the `+` button.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-customer.md:22]:
```suggestion
- At the end of the `Current Access` list will be an input field. Type in the first-, last name or email address to find and select the user you want to add. Choose a Customer Role (Customer Owner, Admin or Employee) and press the `+` button.
```

File: docs/meshstack.how-to.onboard-team-to-customer.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+![Add a user to the customer](../assets/customer/customer-access-control-add-a-user.png "add a user")
+
+- Customer Owner can only be granted to max. 2 user per Customer. Also only a Customer Owner can grant another user the Owner role - except there is no Customer Owner at all.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-customer.md:26]:
```suggestion
- Customer Owner can only be granted to max. 2 users per Customer. Also, only a Customer Owner can grant another user the Owner role - except there is no Customer Owner at all.
```

File: docs/meshstack.how-to.onboard-team-to-project.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+- Select the meshProject using the top navigation bar
+
+![Select meshProject in the upper left corner](../assets/project/project-in-top-nav.png)
+
+- Select the meshproject from the meshCustomer overview
+
+![Select meshProject in meshCustomer overview](../assets/project/project-in-overview.png "Access Control - Access Requests")
+
+- Go to the access control panel by clicking on the `Access Control` tab
+- At the end of the `Current Access` list will be a input field. Type in the first-, last-name or email address to find and select the user you want to add. Choose a Project Role and press the `+` button.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.onboard-team-to-project.md:28]:
```suggestion
- At the end of the `Current Access` list will be an input field. Type in the first-, last name or email address to find and select the user you want to add. Choose a Project Role and press the `+` button.
```

File: docs/meshstack.how-to.manage-partner-level-permissions.md [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+
+If you are not familiar with what a Partner User is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/administration.index.html#docsNav).
+
+## Pre-Requisites
+
+- Permissions: Your user needs the Partner Admin role in the Partner Customer to assign other users administration and/or privileged roles
+
+## Step to Step Guide
+
+1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.manage-partner-level-permissions.md:14]:
```suggestion
1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section by clicking on the "ADMIN AREA" button in the top navigation.
```

File: docs/meshstack.how-to.confirm-project-deletion.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+
+![Click the Archive icon to access the meshProject history](./assets/partner/navigate-to-project-history.png "Access the meshProject history")
+
+- Decline Deletion: You can also decline the meshProject deletion request and bounce it back to the customer e.g. if they forgot to clean up certain ressources in the Cloud Account.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.confirm-project-deletion.md:31]:
```suggestion
- Decline Deletion: You can also decline the meshProject deletion request and bounce it back to the customer e.g. if they forgot to clean up certain resources in the Cloud Account.
```

File: docs/meshstack.how-to.confirm-project-deletion.md [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+
+- Permissions: Your user needs the appropriate privileged role in the Partner account to approve the meshProject deletion. Please check the [Partner Role overview](https://docs.meshcloud.io/docs/administration.index.html#docsNav). Usually meshProject deletions are approved by the Cloud Foundation Team or the Platform Team.
+
+## Step to Step Guide
+
+1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.
+![Select Admin Area in the top navigation bar](./assets/partner/navigate-to-admin-area.png "Go to the Admin Area")
+2. You now need to go to the Project Deleted Projects view where see the overview of meshProjects pending for deletion.
+![Select Project - Deleted Projects menu item on the left side](./assets/partner/navigate-to-project-deletion.png "Deleted Projects")
+3. Confirm the meshProject deletion by clicking on the bin icon. A pop-up for the meshProject deletion will be show a overview of the meshProject and the possibility to add a comment for the meshProject deletion. The checkbox is required to be checked to confirm the deletion.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.confirm-project-deletion.md:18]:
```suggestion
3. Confirm the meshProject deletion by clicking on the bin icon. A pop-up for the meshProject deletion will show an overview of the meshProject and the possibility to add a comment for the meshProject deletion. The checkbox is required to be checked to confirm the deletion.
```

File: docs/meshstack.how-to.confirm-project-deletion.md [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+
+If you are not familiar with what a Partner User is, please check the [official meshcloud documentation](https://docs.meshcloud.io/docs/administration.index.html#docsNav).
+
+## Pre-Requisites
+
+- Permissions: Your user needs the appropriate privileged role in the Partner account to approve the meshProject deletion. Please check the [Partner Role overview](https://docs.meshcloud.io/docs/administration.index.html#docsNav). Usually meshProject deletions are approved by the Cloud Foundation Team or the Platform Team.
+
+## Step to Step Guide
+
+1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section but clicking on the "ADMIN AREA" button in the top navigation.
```
Comment (uleMeshcloud) [inline docs/meshstack.how-to.confirm-project-deletion.md:14]:
```suggestion
1. Login into the meshPanel and navigate to the Administration Area. You can access the Admin section by clicking on the "ADMIN AREA" button in the top navigation.
```

**Review by uleMeshcloud — COMMENTED** [review]
Most of them are link issues and some smaller spelling mistakes

**Comment by felixzieger** [comment]
https://github.com/meshcloud/meshcloud-docs/pull/530 is the follow up for the suggestions that did not make it into this PR

**Pull request merged**